evap/evaluation/tests/test_views.py

```diff
@@ -183,3 +183,36 @@ def test_edit_display_name(self):
 
         page = self.app.get(self.url, user=self.responsible)
         self.assertContains(page, "testdisplayname")
+
+        form = page.forms["profile-form"]
+        form["first_name_chosen"] = "testdisplayname2"
+        form.submit(name="operation", value="illegal", status=400)
+        self.assertFalse(UserProfile.objects.filter(first_name_chosen="testdisplayname2").exists())
+
+
```

Comment:
Changes from another domain unrelated to this PR -- I think something went wrong during merging / rebasing here?